pkg/apis/hco/v1alpha1/hyperconverged_resources.go

```diff
@@ -19,23 +21,27 @@ func (r *HyperConverged) getNamespace(defaultNamespace string, opts []string) st
 	return defaultNamespace
 }
 
-func (r *HyperConverged) getLabels() map[string]string {
+func (r *HyperConverged) getLabels(component hcoutil.AppComponent) map[string]string {
 	hcoName := hcoutil.HyperConvergedName
 
 	if r.Name != "" {
 		hcoName = r.Name
 	}
 
 	return map[string]string{
-		hcoutil.AppLabel: hcoName,
+		hcoutil.AppLabel:          hcoName,
+		hcoutil.AppManagedByLabel: hcoutil.OperatorName,
+		hcoutil.AppVersionLabel:   os.Getenv(hcoutil.HcoKvIoVersionName),
```

Comment:
Also, here you're setting the `HCO_KV_IO_VERSION` version to all operands, although in the CSV you're setting the operand specific version (e.g., CNAO 0.44.0) - need to decide on a consistent approach. If the operand specific version is used, then probably the application name label should reflect that (e.g., `hcoutil.AppLabel: "cnao"`).